src/library/scala/collection/immutable/Map.scala

```diff
@@ -153,6 +153,10 @@ object Map extends ImmutableMapFactory[Map] {
     override def foreach[U](f: ((K, V)) => U): Unit = {
       f((key1, value1))
     }
+    override def exists(p: ((K, V)) => Boolean): Boolean = p((key1, value1))
```

Comment:
I have checked that the changes to this file correspond to those in https://github.com/scala/scala/commit/20b4596bc340ccfc5ba37847d6b21215586f0398#diff-e4f24e9fd33bb1494a43fee25d3b712f